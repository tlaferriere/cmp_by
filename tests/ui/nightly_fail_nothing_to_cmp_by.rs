use cmp_by_derive::CmpBy;

#[derive(CmpBy)]
struct Thing {
    a: u32,
    b: u64,
    f: f64,
}

fn main() {}
