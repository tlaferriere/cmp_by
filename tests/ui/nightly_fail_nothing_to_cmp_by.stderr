error: CmpBy: no field to compare on. Mark fields to compare on with #[cmp_by]
 --> tests/ui/nightly_fail_nothing_to_cmp_by.rs:5:5
  |
5 | /     a: u32,
6 | |     b: u64,
7 | |     f: f64,
  | |___________^
